SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -17320,7 +17320,15 @@ void CNSSolver::SetRoe_Dissipation(CGeometry *geometry, CConfig *config){
     }
     
     if (kind_roe_dissipation == NTS || kind_roe_dissipation == NTS_DUCROS){
-      node[iPoint]->SetRoe_Dissipation_NTS();
+      /*--- XXX: This grid length does not match the original paper.
```

Comment:
Remove XXX?

Reply:
What is the SU2 convention for commenting a "hack"? I wanted to denote that the grid length used here does *not* match the grid length from the original papers.

Reply:
I see.. no standard "hack" convention. Unless you need something searchable, I think your comment is clear enough.